doc/README_docker_gnu.md

```diff
@@ -1,3 +1,5 @@
+# Short-Range Weather App Public Release in Docker
+
 This document explains how to run the SRW public release, version 1,
 in a Docker container. We will run twice: once by building an image,
 and a second time manually.
```

Comment:
>We will run twice: once by building an image, and a second time manually.

I feel that this wording is confusing, especially for users not intimately familiar with both docker *and* the UFS SRW App. I'd suggest changing the wording to something like:

>We will run twice: once by building a Docker image complete with the application, and a second time building the application manually within a pre-existing Docker image.